### PR TITLE
Make run_tests.sh go faster

### PR DESCRIPTION
**Pull request by nedbat**
Use Sphinx features to build faster.  No content changes.
Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] @jmbowman 

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by catong** [comment]
@nedbat My only question is: what's in the edx-sphinx-theme update?

File: run_tests.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ for project in "${projects[@]}"; do
     # -n runs in nit-picky mode.
     # -E Don’t use a saved environment (the structure caching all
     #    cross-references), but rebuild it completely.
-    make html SPHINXOPTS="-E -n -w $err_log_file"
+    make html SPHINXOPTS="-j4 -E -n -w $err_log_file"
```
Comment (jmbowman) [inline run_tests.sh:69]: Should probably explain the `-j4` parameter in the comment above the make command, since the other parameters are covered there.

**Review by jmbowman — APPROVED** [review]

**Comment by jmbowman** [comment]
@catong He had to update the theme extension to explicitly state that it's compatible with parallel builds: https://github.com/edx/edx-sphinx-theme/pull/7

**Comment by catong** [comment]
Thanks @nedbat, @jmbowman 👍

File: run_tests.sh (hunk 1 above)
Comment (nedbat) [reply run_tests.sh:69]: Nice. done.

**Pull request merged**